lib/keychain/keychain.rb

```diff
@@ -102,6 +136,37 @@ def generic_passwords
       Scope.new(Sec::Classes::GENERIC, self)
     end
 
+    # Imports item from string or file to this keychain
+    #
+    # @param [String, File] input String or File with raw data to import
+    # @param [Array <String>] trusted_apps List of applications which will be
+    # permitted to access imported items
+    def import(input, trusted_apps=[])
```

Comment:
Would it be useful if this returned the imported item? Should you accept IO rather than file (thinking of people piping in things) ?